mono/metadata/appdomain.c

```diff
@@ -198,13 +198,15 @@ create_domain_objects (MonoDomain *domain)
 	mono_error_assert_ok (&error);
 	mono_field_static_set_value (string_vt, string_empty_fld, empty_str);
 	domain->empty_string = empty_str;
+	mono_gc_wbarrier_generic_nostore (&domain->empty_string);
```

Comment:
Not sure it matters, but would replacing assignment above with `mono_gc_wbarrier_generic_store` be better?

Reply:
there should be no practical difference

Reply:
Looking at it, I decided not to change it. The functionality is identical, and the code did not look nicer by merging the assignment and the write barrier, as lines become very long. Merged as is.